Makefile

```diff
@@ -0,0 +1,158 @@
+VIRTUALENV=~/.venvs/pixua
+
+.PHONY: setup
+setup: requirements install
+
+.PHONY: requirements
+requirements: requirements_py requirements_js
+	@echo "Installed requirements"
+
+.PHONY: install
+install: requirements install_py install_js
```

Comment:
Actually, as far as I understand, both depend on requirements, but requirements will be run only once.

Considering both depend on requirements and they can be called fairly independently, I would leave the requirement on both and let make sort this out.

Does this make sense?